test/readrows.ts

```diff
@@ -252,6 +252,7 @@ describe('Bigtable/ReadRows', () => {
     // Transform stream
     const transform = new Transform({
       objectMode: true,
+      writableHighWaterMark: 0,
```

Comment:
why is `writableHighWaterMark` only set in the test and not in the implementation (in `src/table.ts`) similar to `readableHighWaterMark` ?